### DISCUSSION TITLE
Unable to pass GitHub Education certification

### DISCUSSION THREAD

**Discussion by vidomicm** [community/community#52222] [GitHub Education]
I have completed the GitHub user profile as requested in the email and uploaded my personal and school ID photos, why can't I pass the verification? (I have tried 5 times to verify)

**Comment by xdouworld** [comment]
May I ask if you have passed now? If so, could you share your experience? Thank you very much

**Reply by vidomicm** [reply]
没，还是没用

**Reply by xdouworld** [reply]
方便留个联系方式吗？一起交流一下。

**Reply by vidomicm** [reply]
2376714571是你的QQ吗？

**Comment by samus-aran** [comment]
Hi @vidomicm 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Let us know how you get on 👍
